packages/aragon-wrapper/src/index.js

```diff
@@ -477,6 +493,92 @@ export default class Aragon {
       .toPromise()
   }
 
+  /**
+   * @param {Array<Object>} An array of Ethereum transactions that describe each step in the path
+   * @return {Promise<string>} transaction hash
+   */
+  performTransactionPath (transactionPath) {
+    return new Promise((resolve, reject) => {
+      this.transactions.next({
+        transaction: transactionPath[0],
+        path: transactionPath,
+        accept (transactionHash) {
+          resolve(transactionHash)
+        },
+        reject (err) {
+          reject(err || new Error('The transaction was not signed'))
+        }
+      })
+    })
+  }
+
+  /**
+   * Performs an action on the ACL using transaction pathing
+   *
+   * @param {string} method
+   * @param {Array<*>} params
+   * @return {Promise<string>} transaction hash
+   */
+  async performACLIntent (method, params) {
+    const aclAddr = this.aclProxy.address
+
+    if (method === 'createPermission') {
```

Comment:
Would it be feasible to filter the acl's artifact for methods that have roles attached instead of hardcoding names?

Reply:
Good catch, we actually can, adding it